### ISSUE TITLE
Caret and selected text not shown in Firefox

### ISSUE THREAD

**Issue by stefanreimers** [Polymer/polymer#4441]
### Description
When using an input tag in a Polymer 2 element, the caret is not visible when focused and selected text is not highlighted. Editing the value of the input or selecting text works fine. This only affects Firefox (at least 52, 54). Does not occur in Chrome, IE11, Edge.
Reason seems to be the _focus_ event handler on _window_, which is set in _webcomponents-hi-sd-ce.js_ on in function _activateFocusEventOverrides_. 

#### Steps to Reproduce
1. Install polymer-cli (see https://www.polymer-project.org/2.0/start/install-2-0)
2. Verify Polymer version 0.18.0-pre.13
3. Initialize project with option _polymer-2-element_
4. Alter created sample element to include an input tag
5. Load page via _polymer serve_

#### Expected Results
Caret should be showing, selected text should be highlighted

#### Actual Results
Caret not visible, selected text not highlighted, input seems to loose focus

### Browsers Affected
- [ ] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Safari 9
- [ ] Safari 8
- [ ] IE 11

### Versions
- Polymer: 2.0-preview
- webcomponents: 1.0.0-rc.5

**Comment by azakus** [comment]
Closing in favor of https://github.com/webcomponents/shadydom/issues/61
